assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/notes-and-events/notes-and-events-view.tsx

```diff
@@ -0,0 +1,167 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { useStyles } from '@Pimcore/modules/asset/editor/shared-tab-manager/tabs/notes-and-events/notes-and-events-view.style'
+import { Button, Input, Select } from 'antd'
+import {
+  type Note
+} from '@Pimcore/modules/asset/editor/shared-tab-manager/tabs/notes-and-events/notes-and-events-api-slice.gen'
+import { Icon } from '@Pimcore/components/icon/icon'
+import { NoteAndEventCard } from '@Pimcore/components/note-and-event-card/note-and-event-card'
+import { formatDateTime } from '@Pimcore/utils/helpers'
+import { useModal } from '@Pimcore/components/modal/useModal'
+import { ModalFooter } from '@Pimcore/components/modal/footer/modal-footer'
+import TextArea from 'antd/es/input/TextArea'
+
+interface NotesAndEventsTabViewProps {
+  notes: Note[]
+  pagination: React.JSX.Element
+  onClickTrash: (id: number) => void
+  onClickSaveNote: (type: string, title: string, description: string) => void
+}
+export const NotesAndEventsTabView = ({
+  notes,
+  pagination,
+  onClickTrash,
+  onClickSaveNote
+}: NotesAndEventsTabViewProps): React.JSX.Element => {
+  const { t } = useTranslation()
+  const { styles } = useStyles()
+  const { renderModal: RenderModal, showModal, handleOk } = useModal()
+
+  let type = ''; let title = ''; let description = ''
+
+  const NotesAndEvents = notes.map((note) => {
+    let showDetails = false
+    const formatedData: any[] = []
+    if (Array.isArray(note.data) && note.data.length > 0) {
+      showDetails = true
+      note.data.forEach((noteData) => {
+        const tempData = structuredClone(noteData)
+        if (typeof tempData.data === 'object') {
+          tempData[t('notes-and-events.value')] = tempData.data.path
+        } else {
+          tempData[t('notes-and-events.value')] = tempData.data
+        }
+        delete tempData.data
+        formatedData.push(tempData)
+      })
+    } else {
+      // TODO
```

Comment:
What is the todo here? 
Should we create an separate issue for that?

Reply:
We can delete this, the type was changed in the backend